From 2c168e76d7616dd0d772e0dd8a7df263cf926749 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Fri, 18 Mar 2016 17:38:16 +0200
Subject: [PATCH 1274/1429] staging: fsl-mc: Export function

The DPAA2 Ethernet driver calls mc_get_version() in order to get the
firmware version for the ethtool driver info. The function needs to be
exported otherwise the Ethernet driver can't compile as a module.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-mc/bus/dpmng.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/dpmng.c b/drivers/staging/fsl-mc/bus/dpmng.c
index 1aeca05..387390b 100644
--- a/drivers/staging/fsl-mc/bus/dpmng.c
+++ b/drivers/staging/fsl-mc/bus/dpmng.c
@@ -58,6 +58,7 @@ int mc_get_version(struct fsl_mc_io *mc_io,
 
 	return 0;
 }
+EXPORT_SYMBOL(mc_get_version);
 
 int dpmng_get_container_id(struct fsl_mc_io *mc_io,
 			   uint32_t cmd_flags,
-- 
1.7.5.4

